docker/README.md

```diff
@@ -54,27 +68,50 @@ The TorchServe's inference and management APIs can be accessed on localhost over
 curl http://localhost:8080/ping
 ```
 
-#### Check running containers
+# Create TorchServe docker image from source
+
+The following are examples on how to use the `build_image.sh` script to build Docker images to support CPU or GPU inference.
+
+To build the TorchServe image for a CPU device using the `master` branch, use the following command:
 
 ```bash
-docker ps
+./build_image.sh
```

Comment:
@dhaniram-kshirsagar The docker dev build command throws many errors for the failing gradle tests while the build is happening

```
 5 tests completed, 1 failed
  
  > Task :server:test FAILED
  > Task :server:jacocoTestReport
  
  FAILURE: Build failed with an exception.
  
  * What went wrong:
  Execution failed for task ':server:test'.
  > There were failing tests. See the report at: file:///serve/frontend/server/build/reports/tests/test/index.html
```
  

However after this it still proceeds and creates a container. 

```
Failed to build torchserve
Installing collected packages: torchserve
    Running setup.py install for torchserve: started
    Running setup.py install for torchserve: still running...
    Running setup.py install for torchserve: still running...
    Running setup.py install for torchserve: finished with status 'done'
Successfully installed torchserve-0.0.1b20200510
Removing intermediate container a7e2317f538d
 ---> b30ed2bfdf4f
Step 9/20 : RUN useradd -m model-server     && mkdir -p /home/model-server/tmp
....

```

Please fix the errors so that all the gradle tests work fine which will be needed for the development build.

Note: The build for branch issue_266 worked fine, so unclear where the error is.
`./build_image.sh -b issue_266 -t torchserver:dev` gave no errors

Reply:
I tried to reproduce this with latest changes however no luck. IMO, fixes #352 and #341 should take care of these intermittent build failures.

Reply:
Again tried different combinations to repro this however still no luck. With changes in this PR [branch -> issue_266] 

cd docker
./build_image.sh [checkout master]
--> successfully created image

- introduce some error in any of java file [to fail the build]
- modify the build_image.sh so that it uses existing serve
-finally execute script
./build_image.sh
--> build fails and no docker file is created.

Please provide details if the above steps are incorrect.

Reply:
Verified that this works on branch `issue_266`